pom.xml

```diff
@@ -160,6 +155,12 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter</artifactId>
+            <exclusions>
```

Comment:
I think it would make sense to have the different dependencies guarded with a profile?

Perhaps have a default profile that uses the new jakarta ee deps and an "javax" profile, which uses the old dependencies?

Reply:
An interesting thought, normally I do write services that follow one simple ruleset which perfectly fits the surrounding services. Frameworks with multiple supported versions are completely new for me. ;)